go.mod

```diff
@@ -260,6 +260,7 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.6.3
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.6.3
 	gocloud.dev v0.25.0
+	gotest.tools v2.2.0+incompatible
```

Comment:
🤔 Do we use this? I don't see where 🙈

Reply:
I had to run go mid tidy, because of enterprise, I am not sure why

Reply:
https://github.com/grafana/grafana-enterprise/pull/4044/files

Reply:
I looked it up, it looks like it's an indirect dependency 🤷 I'm not sure why it doesn't show up as such, but eh :+1:

Reply:
interesting 🤷‍♀️